srcalc/src/main/java/gov/va/med/srcalc/domain/calculation/ValueRetriever.java

```diff
@@ -224,6 +224,33 @@ public String retrieveReferenceInfo(final Patient patient, final Variable var)
             }
             return variableRefInfo.toString();
         }
+    },
+    POTASSIUM
+    {
+        @Override
+        public void execute(final Patient patient, final VariableEntry variableEntry, final Variable variable,
+            final String key)
+        {
+            addLabValue("POTASSIUM", patient, variableEntry, key);
```

Comment:
The unit tests only run `ValueRetriever.WBC` right now. I think we should update `testWithRetrievedLabs()` to test "retrieving" all supported labs.